Resources/Prototypes/Entities/Constructible/Walls/signs.yml

```diff
@@ -0,0 +1,793 @@
+# These signs would not have collision otherwise, they simply
+# need collision in order for them to be destructible. Once the
+# SmallImpassable etc. stuff actually gets implemented in this
+# shitheap of a videogame bother swept to go back and fix these.
+
+- type: entity
+  id: PaintingMonkey
+  name: monkey painting
+  components:
+  - type: Clickable
+  - type: InteractionOutline
+  - type: Collidable
+    shapes:
+    - !type:PhysShapeAabb
+      bounds: "-0.1,-0.4,0.1,0.4"
+      layer:
+      - Impassable
+      - MobImpassable
+      - VaultImpassable
+      - SmallImpassable
+  - type: Damageable
+  - type: Destructible
+    thresholdvalue: 5
+  - type: Sprite
+    drawdepth: WallTops
+    sprite: Constructible/Misc/decals.rsi
+    state: monkey_painting
+  - type: Icon
+    sprite: Constructible/Misc/decals.rsi
+    state: monkey_painting
```

Comment:
Maybe use a common parent type to avoid repeating at least `Damageable`, `Destructible`, `Clickable`, `InteractionOutline`.